draft-ietf-ppm-dap.md

```diff
@@ -788,11 +795,12 @@ Before the Client can upload its report to the Leader, it must know the HPKE
 configuration of each Aggregator. See {{compliance}} for information on HPKE
 algorithm choices.
 
-Clients retrieve the HPKE configuration from each aggregator by sending an HTTP
-GET request to `{aggregator}/hpke_config`. Clients MAY specify a query parameter
-`task_id` whose value is the task ID whose HPKE configuration they want. If the
-Aggregator does not recognize the task ID, then it MUST abort with error
-`unrecognizedTask`.
+Clients retrieve the HPKE configuration from the Leader by sending an HTTP GET
+request to `{leader}/hpke_config`. Similarly, Clients retrieve the HPKE
+configuration from the Helper by sending an HTTP GET request to
+`{helper}/hpke_config`. Clients MAY specify a query parameter `task_id` whose
```

Comment:
It is a bit odd to have this structure.  I would have said global = `{aggregator}/hpke_config` and task-specific = `{aggregator}/tasks/{task-id}/hpke_config`.  Though I suspect that achieving consistency would be harder for the latter form, if you ever had to worry about that.

Reply:
Nice idea! I'll create an issue to track.